woodwork/serializers/parquet_serializer.py

```diff
@@ -26,6 +39,69 @@ def serialize(self, dataframe, profile_name, **kwargs):
         return super().serialize(dataframe, profile_name, **kwargs)
 
     def write_dataframe(self):
```

Comment:
nit: Can we rename this function since it isn't actually writing the dataframe now? Perhaps `make_pyarrow` or something of the sort would be more in line with the method

Reply:
Also docstrings would be helpful here! I realize these methods were already missing docstrings before, but would be great to add them or file an issue to track adding them.

Reply:
Regarding renaming the function - it's a little complicated. If you look at the code flow we first call the `serialize` method on this class, then the `serialize` method on the base class, and that ends up calling `save_to_local_path` (also from the base class) which then calls this `write_dataframe` method which overrides the base class method.

So, if we renamed this we would have to do one of two things:
1. Override the `save_to_local_path` method on this subclass to call the renamed method instead of `write_dataframe`
2. Leave a `write_dataframe` method here that simply calls the renamed method.

If we simply renamed this, when `Serializer` base tries to call `write_dataframe` we would get a `NotImplementedError` because the base class doesn't implement this method and neither would the subclass in this case.

I could also add a note here to explain why this behavior is different for this serializer and leave things as is.

Thoughts on any of these options?

Reply:
Ended up going with option 1 and renamed a couple other methods to make things more clear. Added docstrings for those unique methods as well.